desktop/apps/artwork/components/related_artists/index.coffee

```diff
@@ -24,11 +25,11 @@ setupFollowButtons = ({ $el, relatedArtists, kind }) =>
   following?.syncFollows ids
 
 render = ({ relatedArtists, showMore, baseHref}) ->
-  viewAllUrl = sd.CLIENT.artists[0].href + '/related-artists'
+  viewAllUrl = get(sd.CLIENT, 'artists.0.href' + '/related-artists', '')
```

Comment:
👌  works for me! Since this is coffeescript I have to mention that the existential operator is a bit nicer `sd.CLIENT?.artists?[0]?.href`

Reply:
Its so funny - thats the most useful thing but I forget about it 100% of the time